.github/workflows/test-production-build.yml

```diff
@@ -40,7 +43,45 @@ jobs:
           cp ${GITHUB_WORKSPACE}/tests/fixtures.drainpipe-test-build/composer.json .
           cp ${GITHUB_WORKSPACE}/tests/fixtures.drainpipe-test-build/Taskfile.yml .
           composer update ${{ matrix.composer-flags }}
-      - name: Run static tests
+      - name: Build a production Drupal site
         run: |
           cd ../drupal
           ./vendor/bin/task build
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '18'
+      - name: Install Tugboat CLI for validation
+        run: |
+          cd /usr/local/bin
+          curl -O https://dashboard.tugboatqa.com/cli/linux/tugboat.tar.gz
+          tar xzvf tugboat.tar.gz
+      - name: Validate Tugboat file generation
```

Comment:
These could be turned into matrix parameters in the future, but it's fast now and saves burning CI time on the setup steps.